src/dialect/mssql/mssql-dialect-config.ts

```diff
@@ -0,0 +1,34 @@
+import { ConnectionPool } from 'mssql'
+import { Request, TYPES } from 'tedious'
+import { DatabaseConnection } from '../../driver/database-connection.js'
+
+export interface MssqlDialectConfig {
+  /**
+   * A mssql Pool instance or a function that returns one.
+   *
+   * If a function is provided, it's called once when the first query is executed.
+   *
+   * https://github.com/sidorares/node-mysql2#using-connection-pools
```

Comment:
note to self, refer to the `mssql` connection pools readme section, not `mysql2`'s.